### ISSUE TITLE
[Accessibility] Inspect: Cannot find toolStripScrollButtons in hierarchy of toolStripDropDown menu

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#6652]
* .NET Core Version: 
7.0.100-preview.2.22103.2

* Have you experienced this same bug with .NET Framework?: 
Yes

**Repro steps:**
1. Create a Winforms .Net Core application
2. Add a toolStrip on the form designer
3. Add a toolStripDropDownButton on the toolStrip
4. Add several drop down items under toolStripDropDownButton
5. In form1.cs code file, add 1 line code to define the height of the this.toolStripDropDownButton1.DropDown.MaximumSize property in order to show toolStripScrollButton in the dropdown menu

Or using attached app:
[WinFormsApp94.zip](https://github.com/dotnet/winforms/files/8030101/WinFormsApp94.zip)

6. Build and run app, then using Inspect detects these toolStripScrollButtons in the toolStripDropDownButton dropdown menu

**Actual behavior:**
In inspect, cannot find toolStripScrollButtons in hierarchy of toolStripDropDown menu, and In Narrator, it cannot focus on toolStripScrollButtons to announce.
![image](https://user-images.githubusercontent.com/26474449/153136119-f266d560-54d6-4cdf-bfda-7d81f4ddf66c.png)

**Expected behavior:**
In inspect, toolStripScrollButtons are in hierarchy of toolStripDropDown menu, and Narrator can focus on them to announce.

**Comment by merriemcgaw** [comment]
@Olina-Zhang  what do you see in AI4Win? I think we should switch to that tool because that's what we have to be correct in.

**Comment by Olina-Zhang** [comment]
@merriemcgaw  The result in Accessibility Insight is same as in Inspect:
![image](https://user-images.githubusercontent.com/26474449/154599665-e348827e-90bf-434c-8957-4da6cd6a2fce.png)

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @merriemcgaw, @Tanya-Solyanik, @dreddy-work. 

I have a question about the properties that a `ToolStripScrollButton's` accessibility object should have.
![image](https://user-images.githubusercontent.com/23376742/159866659-9d8cb09a-3352-4556-876b-40e00657c45f.png)

We call it a `ToolStripScrollButton`, although in fact the `StickyLabel` is responsible for its operation. In addition, the user, although he cannot set the keyboard focus, can click on the `ToolStripScrollButton`, which allows you to scroll through the items in the menu.

Therefore, I would like to clarify what kind of control the `ToolStripScrollButton's` accessibility object should be and whether it should have a value for the `DefaultAction`

**Comment by merriemcgaw** [comment]
I like `ToolStripScrollButton` for the name just fine. The default action is to scroll down to make more stuff visible and that doesn't seem to really have a default action at least in the sense of executing some code that the developer sets like the `Click` Event of a `Button`. @Tanya-Solyanik and I are heads down this week on a high priority project. We should take a look at this next week.

**Comment by MelonWang1** [comment]
Verified this issue on .NET 7.0.100-preview.5.22256.2, issue was fixed. In inspect, toolStripScrollButtons are in hierarchy of toolStripDropDown menu.
Inspect:
![Screenshot 2022-05-06 200533](https://user-images.githubusercontent.com/94418985/167238703-c5e0eee4-fb82-484c-ae7b-83f0e872b578.png)
![UP](https://user-images.githubusercontent.com/94418985/167238707-c3661385-ba39-4dc6-b36c-4396457a42c6.png)
Accessibility:
![Screenshot 2022-05-06 213824](https://user-images.githubusercontent.com/94418985/167238722-216a829c-12f5-4dc6-91c3-24e83bfd8819.png)
